javascript/javascript2/week2/index2.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="stylesheet" type="text/css" href="main.css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Open+Sans:400,800"
+      rel="stylesheet"
+    />
+    <title>hyfBayHelpers</title>
+  </head>
+  <body>
+    <h1>HYF BAY</h1>
+    <main class="app">
+      <div class="input-boxes">
+        <h2>Our Products</h2>
+        <form class="form form-input">
+          <input id="searchInput" type="text" class="search-input" />
+          <input id="maxPrice" type="text" name="" class="max-price-input" />
```

Comment:
```suggestion
          <input id="maxPrice" type="number" name="" class="max-price-input" />
```
It's always nice to be specific in which kind of input you're asking from the user.

Reply:
Of course : )  Thank you for pointing that out.